javascript-algorithms-and-data-structures/learn-basic-string-and-array-methods-by-building-a-music-player-app/script.js

```diff
@@ -1,384 +1,265 @@
- //  Songs array creation
- const songs = [
+const playlistSongs = document.getElementById("playlist-songs");
+const playButton = document.getElementById("play");
+const pauseButton = document.getElementById("pause");
+const nextButton = document.getElementById("next");
+const previousButton = document.getElementById("previous");
+const shuffleButton = document.getElementById("shuffle");
+
+const cover =
```

Comment:
Why is this cover variable here?
It doesn't look like it is being used anywhere, is it?

My understanding is that we have the image in the HTML and it doesn't change. So it looks like we don't need it in the JS too.